assets/pages/new-contributions-landing/components/SetPassword/setPassword.scss

```diff
@@ -0,0 +1,35 @@
+.set-password {
+  .form__input--email {
+    pointer-events:none;
+    background-color: #dcdcdc;
+  }
+
+  .form__field--contribution-password, .form__submit--no-thanks{
+    margin-top: $gu-v-spacing;
+  }
+
+  .form__submit {
+    padding-bottom: 0;
+  }
+
+  .form__submit-button {
+    width: 100%;
+    justify-content: space-between;
```

Comment:
I meant to put this on top-level `.form__submit-button` style - then you don't need it here or anywhere else you want the button to be 100%